From c6f7bb39e7f6fd0b29a69eb0b5d8ab2096f00a62 Mon Sep 17 00:00:00 2001
From: Florian Klink <flokli@flokli.de>
Date: Mon, 30 Mar 2020 17:29:54 -0700
Subject: [PATCH 1297/1852] drm: xlnx: fix some typos in the Kconfig help texts

commit ef94c73ce8e1a789f1f76db7c15b67417bcd1162 from
https://github.com/Xilinx/linux-xlnx.git

Fix some typos in the Kconfig help texts.

Signed-off-by: Florian Klink <flokli@flokli.de>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpu/drm/xlnx/Kconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/xlnx/Kconfig b/drivers/gpu/drm/xlnx/Kconfig
index ecc5f90caf61..5463d4aa0302 100644
--- a/drivers/gpu/drm/xlnx/Kconfig
+++ b/drivers/gpu/drm/xlnx/Kconfig
@@ -41,7 +41,7 @@ config DRM_ZYNQMP_DPSUB
 	  DRM KMS driver for ZynqMP DP Subsystem controller. Choose
 	  this option if you have a Xilinx ZynqMP SoC with DisplayPort
 	  subsystem. The driver provides the kernel mode setting
-	  functionlaities for ZynqMP DP subsystem.
+	  functionalities for ZynqMP DP subsystem.
 
 config DRM_XLNX_DSI
 	tristate "Xilinx DRM DSI Subsystem Driver"
@@ -59,7 +59,7 @@ config DRM_XLNX_MIXER
 	depends on DRM_XLNX
 	select VIDEOMODE_HELPERS
 	help
-	  DRM driver for Xilinx Mixer driver
+	  DRM driver for Xilinx Mixer driver.
 
 config DRM_XLNX_PL_DISP
 	tristate "Xilinx DRM PL display driver"
-- 
2.31.1

